apps/taxes/src/modules/avatax/maps/address-map.test.ts

```diff
@@ -0,0 +1,64 @@
+import { describe, expect, it } from "vitest";
+import { avataxAddressFactory } from "./address-map";
+
+describe("mapChannelAddressToAvataxAddress", () => {
+  it("returns fields in the expected format", () => {
```

Comment:
for clarity, try to use class/object name in root describe, then methods can be nested describes. This one is outdated.